en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,267 @@
+Manage assets with grunt
+========================
+
+Context
+-------
+
+To automate all the assets management process, the Open Orchestra Back Office uses the task manager
+``Grunt``_.
+
+*Grunt* is looking for its configuration inside the ``Gruntfile.js`` file in the root directory of
+the project. This file mainly parses external configuration files, split in grunt tasks and grunt
+tasks options.
+
+In order to simplify the management of external bundles assets, these bundles describe their own
+grunt task and/or task options. This way, when adding a new bundle to your project, all you have to do is
+to load these configurations in the ``gruntfile.js`` of your project.
+
+In this documentation, you will see:
+
+- how to _`configure your project to use grunt`
+- how to _`add an external bundle using grunt` tasks / options
+- how to _`add your specific tasks / options to your bundles`
+
+.. _configure your project to use grunt_:
+
+Configuring your project to use grunt
+-------------------------------------
+
+When you install your Open Orchestra Back Office, you need to configure grunt to manage the assets
+contained in the bundles you choose. To do this, you have to create a ``Gruntfile.js``. This file
```

Comment:
i don't like To do this. you can reverse the sentence : you have to create a Gruntfile.js to configure grunt to manage the assets.